tools/paas_dashboard/dashboards/paas-overview.erb

```diff
@@ -0,0 +1,19 @@
+<script type='text/javascript'>
+$(function() {
+  Dashing.numColumns = 3
+  Dashing.widget_base_dimensions = [600, 500]
+});
+</script>
+<div class="gridster">
+  <ul>
+    <li data-row="1" data-col="1" data-sizex="1" data-sizey="1">
+      <a href="https://app.datadoghq.com/monitors#manage?query=service:prod_monitors"><div data-title="Prod" data-id="prod_counts" data-view="Health"></div></a>
+    </li>
+    <li data-row="1" data-col="2" data-sizex="1" data-sizey="1">
+      <a href="https://app.datadoghq.com/monitors#manage?query=service:staging_monitors"><div data-title="Staging" data-id="staging_counts" data-view="Health"></div></a>
+    </li>
+    <li data-row="1" data-col="3" data-sizex="1" data-sizey="1">
+      <a href="https://app.datadoghq.com/monitors#manage?query=service:master_monitors"><div data-title="CI" data-id="ci_counts" data-view="Health"></div></a>
```

Comment:
Could we add `target="_blank"` attribute to the `a` tag?

IMO, it's easier to navigate, if the link would open in a new tab, instead of taking the current state.